### PR TITLE
Improve human-readable output

### PR DESCRIPTION
**Pull request by chmeliik**
- By default, Checkton now uses `sarif-fmt` to display violations
  - Note that `sarif-fmt` also outputs the wiki link for each violation :tada:
- A `display-style` option to allow switching back to `csgrep` to display violations

### Checklist

* [x] Add/update tests for your changes
* [x] Update CHANGELOG.md if your changes are relevant to users (<https://keepachangelog.com/en/1.1.0/#how>)

### CODE REVIEW

**Comment by chmeliik** [comment]
@mmalina turns out adding links to the output wasn't that hard after all \o/

Screenshot from 2024-07-19 16-33-06

**Comment by chmeliik** [comment]
Just need to add a test case for "no violations", since I now see I broke that case :smile:

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@
           image_without_digest=${image%@*}
           version=${image_without_digest##*:}
 
-          if [[ "$tag" != "$version" ]]; then
+          if [[ "$tag" != $version ]]; then
```
[the remark this answers is not in the document]
Comment (chmeliik) [reply .github/workflows/release.yaml]: Fixed

**Comment by chmeliik** [comment]
> @mmalina turns out adding links to the output wasn't that hard after all \o/
> 
> Screenshot from 2024-07-19 16-33-06

Released in v0.2.0

**Comment by mmalina** [comment]
> @mmalina turns out adding links to the output wasn't that hard after all \o/
> 

Awesome, thanks! I added a comment to our Jira about this: https://issues.redhat.com/browse/RELEASE-1024

**Pull request merged**